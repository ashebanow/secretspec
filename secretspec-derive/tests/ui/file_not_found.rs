use secretspec_derive::define_secrets;

// This should fail because the file doesn't exist
define_secrets!("this/file/does/not/exist.toml");

fn main() {}