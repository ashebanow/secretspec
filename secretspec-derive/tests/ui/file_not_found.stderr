error: Failed to parse TOML: I/O error: Failed to resolve path $WORKSPACE/target/tests/trybuild/secretspec-derive/this/file/does/not/exist.toml: No such file or directory (os error 2)
 --> tests/ui/file_not_found.rs:4:1
  |
4 | define_secrets!("this/file/does/not/exist.toml");
  | ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  |
  = note: this error originates in the macro `define_secrets` (in Nightly builds, run with -Z macro-backtrace for more info)
